tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+import django
+from django.conf import settings
+
+
+def pytest_configure():
+    test_db = os.environ.get('TEST_DB', 'sqlite')
```

Comment:
In my opinion makes sense having `sqlite` as the default database but would be nice to add this error `E   django.db.utils.OperationalError: unrecognized token: ":"` in our [contributing guide](https://model-mommy.readthedocs.io/en/latest/#contributing) at some point.

Reply:
I just removed psycopg2 from dev dependency (it is anyways going to be installed via tox). So it should not be a problem for new users.

We might rethink the whole concept of dev_requirements.txt and if we still need it.